packages/core/tests/integration/components/navi-visualizations/apex-gauge-test.js

```diff
@@ -0,0 +1,100 @@
+import { A as arr } from '@ember/array';
+import { set } from '@ember/object';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | apex-gauge ', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('apex gauge at 70%', async function(assert) {
+    assert.expect(3);
+    _setModel(this, 'pageViews', 70000000000);
+    set(this, 'metric', { metric: 'pageViews', paramters: {} });
+    await render(hbs`
+    {{navi-visualizations/apex-gauge
+        model=model
+        options=(hash
+          baselineValue=0
+          goalValue=100000000000
+          metric=metric
+        )
+      }}
+    `);
+    // checks correct color: red = #f05050 = rgba(240, 80, 80, X)
+    assert.equal(
+      this.element.querySelector('.apexcharts-radial-series path').getAttribute('stroke'),
```

Comment:
Can we use QUnit DOM assertions where possible?
https://github.com/simplabs/qunit-dom